### ISSUE TITLE
Missing code line in "Writing your first Django app, Part 4"

### ISSUE THREAD

**Issue by Sebi2020** [django/djangoproject.com#860]
The Index generic view is missing it's model declaration.
Look here: https://docs.djangoproject.com/en/2.2/intro/tutorial04/#amend-views

**Comment by timgraham** [comment]
It's not required because `get_queryset` is defined.

This is the issue tracker for the djangoproject.com website. Please see [TicketClosingReasons/UseSupportChannels](https://code.djangoproject.com/wiki/TicketClosingReasons/UseSupportChannels) for ways to get help with Django usage. Documentation bugs should be reported on [Trac](https://code.djangoproject.com/).

**Comment by Sebi2020** [comment]
No it semms it is required. This is stated by the documentation one paragraph later and Django (2.2) throws an exception if I don't specify one. Even if get_queryset is defined.

**Comment by timgraham** [comment]
The code works for me. Are you sure you haven't made a typo? What does the error say? If you haven't overridden `get_queryset()` properly then you'll see, "IndexView is missing a QuerySet. Define IndexView.model, IndexView.queryset, or override IndexView.get_queryset()."

**Comment by Sebi2020** [comment]
Okay, thats pretty strange. Earlier I got an exception saying that this url (`{% url 'polls:detail' question.id %}` ) cannot be reversed, because there is no view "polls:detail". I haven't changed anything except the views.py file. After adding model=Question to IndexView this exception disappeard and stays gone even if I remove the model field. Really strange.

*BTW*: Maybe People should be guided to the trac bugtracker for documentation issues. It was nowhere an entry for this. Maybe I missed a site, but I only found links for Django issues, djangoproject.com issues and security issues. Since the documentation is served under the djangoproject.com domain I thought this would be the approquiate bug tracker.
